base/Dockerfile

```diff
@@ -100,6 +100,18 @@ RUN apk add --no-cache \
     && apk del .build-dependencies \
     && rm -rf /usr/src/arp-scan
 
+# PicoTTS - it has no specific version - unfortunately we'll have to use master directly
+RUN apk add automake autoconf git libtool popt-dev git build-base \ 
+    && mkdir -p -m 755 /usr/src/pico \

```

Comment:
This folder is created by the git clone below.

Reply:
The update should solve this.